ftw/solr/tests/test_converters.py

```diff
@@ -71,6 +71,26 @@ def test_invalid_date_converts_to_none(self):
         self.assertEqual(to_iso8601('30'), None)
         self.assertEqual(to_iso8601(30), None)
 
+    def test_zope_datetime_high_millis_does_not_cause_rounding_error_low(self):
+        dt = to_iso8601(DateTime('2017-10-21 16:28:16.999501'))
```

Comment:
I'd actually rather use `59.999501` in the test, makes it clear that the issue is that we do not want to round to 60...

Reply:
thanks, that's what i wanted to do, but seems i had a brain freeze 😓